src/pe/options.rs

```diff
@@ -8,14 +9,35 @@ pub struct ParseOptions {
     /// memory](https://learn.microsoft.com/en-us/windows/win32/debug/pe-format#other-contents-of-the-file).
     /// For on-disk representations, leave as true. Default: true
     pub parse_attribute_certificates: bool,
+    /// Whether or not to end with an error in case of incorrect data or continue parsing if able. Default: ParseMode::Strict
+    pub parse_mode: ParseMode,
 }
 
-impl ParseOptions {
+#[derive(Debug, Copy, Clone)]
+pub enum ParseMode {
+    /// Always end with error on incorrect data
+    Strict,
+    /// Incorrect data will not cause to end with error if possible
+    Permissive,
+}
+
+impl Default for ParseOptions {
     /// Returns a parse options structure with default values
-    pub fn default() -> Self {
+    fn default() -> Self {
         ParseOptions {
             resolve_rva: true,
             parse_attribute_certificates: true,
+            parse_mode: ParseMode::Strict,
+        }
+    }
+}
+
+impl ParseOptions {
+    pub fn te() -> Self {
+        Self {
```

Comment:
more idiomatic would be something like:

```rust
Self {
  resolve_rva: false,
  parse_attribute_certificates: false,
  .. Self::default()
}
```

this way if new methods are added don't (maybe) need to update this location in source, but it's fine. I'm also wondering if we should make this non `pub`, just to reduce the api surface for now, i can fix this up though if you don't feel like pushing again.